provisioner/template.go

```diff
@@ -283,6 +284,10 @@ func list(args ...interface{}) []interface{} {
 	return args
 }
 
+func strAppend(list []string, item string) []string {
```

Comment:
Does it make sense to have vararg items like:
```go
func strAppend(list []string, item string, other ...string) []string
```
?

Not sure if that works but we can have a test :)

Reply:
Updated!

Reply:
I suggested to have item+other to prohibit `{{ append .Values.data.items }}` which makes no sense.

In go stdlib they missed that for min/max functions, see https://github.com/golang/go/issues/66179#issuecomment-1989029221

Reply:
Makes sense, updated!